source/reference/minio-mc/mc-idp-ldap-accesskey-info.rst

```diff
@@ -0,0 +1,107 @@
+.. _minio-mc-idp-ldap-accesskey-info:
+
+==============================
+``mc idp ldap accesskey info``
+==============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+.. mc:: mc idp ldap accesskey info
+
+
+Description
+-----------
+
+.. start-mc-idp-ldap-accesskey-info-desc
+
+The :mc:`mc idp ldap accesskey info` outputs information about the specified access key(s).
+
+.. end-mc-idp-ldap-accesskey-info-desc
+
```

Comment:
@taran-p Can you confirm Ravind's assumption?

Reply:
I believe as of right now it will work on any access keys, although that may change in the future and is not intended use

Reply:
Added verbiage.

Reply:
Hm. A bit dicey since this can return all access keys, but it occurs to me that when AD/LDAP is enabled we disable all other IDPs, including MinIO internal. So this is true by virtue of current functionality anyways. So we are good.